src/machine/board_lorae5.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 const (
-	LED1 = PB5
-	LED  = LED1 // Default LED
+	// We assume a LED is connected on PB5
+	LED = PB5 // Default LED
```

Comment:
It looks like this change is unrelated to the PR. Was it intentional?

Reply:
This board don't actually have a LED. 
So I thought I would be a good idea to remove it. 

But, that breaks blinky1 , so I rolled-back the change.

Reply:
FWIW in the LoRa-e5 schematic, PB5 is connected to a RED LED through a jumper (see bottom right of the schematic): https://files.seeedstudio.com/products/113990934/LoRa-E5%20Dev%20Board%20v1.0.pdf

It certainly blinked the LED for me :)

Reply:
![image](https://user-images.githubusercontent.com/561441/143569811-d76a11e2-854c-4d92-8ac9-b910bf1507b3.png)

I understand why i've never seen this led blinking and why I connected an external LED (hopefully on PB5)

Reply:
:-) the official dev board does look a little less artisanal.